README-developers.md

```diff
@@ -130,3 +127,16 @@ argument makes git modify the latest commit in the local branch.
 
 See more details about modifying history from
 [Git Tools - Rewriting History](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)
+
+#### Clean up after your changes are merged into the master branch
+
+After you merge your pull, GitHub will give you the option to delete your branch
+from the GitHub repository. Usually you will want to do so. Then, to delete your
+local branch and bring all the latest changes into your local repository:
+
+```
+git checkout master
+git branch -D my_cool_new_feature
+git pull --ff-only
```

Comment:
If you first fetch or pull the latest changes then you can use lowercase `-d` argument.

There is `--merged` arguent. Which ca be used to list merged local branch `git branch --merged origin/master`

Reply:
1. I prefer not to wrap lines. That is a task better left to a computer. My editor likely has a setting to do it automatically, but I've not yet discovered it. In wrapping here, I was just following the style of the rest of the document. I'm not sure what you are suggesting. Do you prefer lines wrapped or unwrapped?

2. What should the sequence be? I tried to use `-d`, but was unsuccessful.

Reply:
> 1. I prefer not to wrap lines. That is a task better left to a computer. My editor likely has a setting to do it automatically, but I've not yet discovered it. In wrapping here, I was just following the style of the rest of the document. I'm not sure what you are suggesting. Do you prefer lines wrapped or unwrapped?

I don't ind either. If we use unwrapped, then I have to set a project specific default to disable the wrap.

> 2. What should the sequence be? I tried to use `-d`, but was unsuccessful.

`-d` should work if your branch had been merged. Snippet from `git branch --help`
```
-d, --delete
Delete a branch. The branch must be fully merged in its upstream branch, or in HEAD if no upstream was set with --track or --set-upstream-to.
```

Reply:
1. I see now! I did not realize that I had left some lines unwrapped. I'll go wrap them.

2. Git would not let me delete the local branch with `-d` and the remote branch had already been deleted. What ought I to have done to avoid that situation? Having reached it, what should I have done next? Does "The branch must be fully merged" mean that I must merge changes from the remote branch (or HEAD) into the local branch? It seems foolish to do so only to then delete the branch. It reminds me of the sometime US practice of refurbishing military bases before demolishing them.

Reply:
Wrapped lines pushed.

Reply:
> 2. Git would not let me delete the local branch with `-d` and the remote branch had already been deleted. What ought I to have done to avoid that situation? Having reached it, what should I have done next? Does "The branch must be fully merged" mean that I must merge changes from the remote branch (or HEAD) into the local branch? It seems foolish to do so only to then delete the branch. It reminds me of the sometime US practice of refurbishing military bases before demolishing them.

The remote tracking is the issue. It won't work always. Git might be looking for merge from a wrong remote branch.  Better forget about lower case argument.